integration_tests/src/main/python/delta_lake_update_test.py

```diff
@@ -72,7 +72,8 @@ def checker(data_path, do_update):
                           delta_writes_enabled_conf  # Test disabled by default
                           ], ids=idfn)
 @pytest.mark.skipif(is_before_spark_320(), reason="Delta Lake writes are not supported before Spark 3.2.x")
-def test_delta_update_disabled_fallback(spark_tmp_path, disable_conf):
+@pytest.mark.parametrize("deletion_vector_conf", deletion_vector_conf, ids=idfn)
+def test_delta_update_disabled_fallback(spark_tmp_path, disable_conf, deletion_vector_conf):
```

Comment:
Why do we need deletion_vector_conf for this at all? We should still fall back on a write with deletion_vectors enabled?